integration/integration_test.sh

```diff
@@ -27,32 +26,17 @@ echo "Event Counts:"
 echo " - Good: ${good_count}"
 echo " - Bad: ${bad_count}"
 
-stream_enrich_pid_file=/var/run/snowplow_stream_enrich_0.10.0.pid
```

Comment:
Why remove all the pid testing to check the service restarts?

Reply:
They are moved to control plane test suite under the control plane folder.